services/apps/alcs/src/alcs/notification/notification.service.ts

```diff
@@ -45,9 +45,7 @@ export class NotificationService {
     });
   }
 
-  async createNotificationForApplication(
-    notificationDto: CreateNotificationServiceDto,
-  ) {
+  async saveNotification(notificationDto: CreateNotificationServiceDto) {
```

Comment:
Just a nitpick of nitpicks, but I would prefer something like add/create notification rather than save

Reply:
Done!